### PR TITLE
Refactor tests to pest

### PR DESCRIPTION
**Pull request by AyoobMH**

### CODE REVIEW

File: tests/ArgumentConverterTest.php [hunk 1]
```diff
@@ -1,36 +1,27 @@
...
+});
 
-    /** @test */
-    public function it_returns_int_values_unchanged()
-    {
-        $this->assertEquals(1, ArgumentConverter::convertToPrimitive(1));
-        $this->assertEquals(PHP_INT_MAX, ArgumentConverter::convertToPrimitive(PHP_INT_MAX));
-    }
+it('returns int values unchanged', function () {
+    assertEquals(1, ArgumentConverter::convertToPrimitive(1));
```
Comment (freekmurze) [inline tests/ArgumentConverterTest.php:15]: Could you rewrite all assertions to `expect()` ?

**Review by freekmurze — CHANGES_REQUESTED** [review]

File: tests/ArgumentConverterTest.php (hunk 1 above)
Comment (AyoobMH) [reply tests/ArgumentConverterTest.php:15]: Hi @freekmurze, I'm planning to open more PRs, should I do it for the next PRs as well? Thanks!

Comment (freekmurze) [reply tests/ArgumentConverterTest.php:15]: Yup!

**Comment by freekmurze** [comment]
Thank you!

**Pull request merged**